bedrock/thunderbird/templates/thunderbird/start/release.html

```diff
@@ -34,6 +72,9 @@ <h2 class="iconic">{{ _('Need Support?') }}</h2>
           <ul>
             <li><a href="{{ url('thunderbird.features') }}">{{ _('Features') }}</a></li>
             <li><a href="https://blog.mozilla.org/thunderbird/">{{ _('Visit our blog') }}</a></li>
+            {% if l10n_has_tag('thunderbird_startpage_donate_2016_06') %}
+            <li><a href="https://donate.mozilla.org/thunderbird/about/">{{ _('Donate') }}</a></li>
```

Comment:
We can avoid the tag in this case, since "Donate" is already available as a string in shared files.